assets/scss/settings/fonts.scss

```diff
@@ -0,0 +1,11 @@
+$georgia: Georgia;
+$palatino: Palatino;
+$times_New_Roman: Times;
```

Comment:
Name of variable `times_New_Roman` should be written in all lowercase letters with hyphens instead of underscores